src/GCRealTimeMon/Utilities/SpectreConsoleOut.cs

```diff
@@ -126,6 +122,36 @@ public void WriteStatsUsage()
             WriteRule($"[{ThemeConfig.Current.MessageColor}]press [{ThemeConfig.Current.HighlightColor}]s[/] for current stats or any other key to exit[/]");
         }
 
+        public async Task PrintCallStack(TraceCallStack callstack, string eventName)
+        {
+            await liveOutputTable.StopAsync();
+
+            WriteRule();
```

Comment:
If both plaintext & spectre now have a use case for writing line separators, it would be nice to add it to the interface and use a common name. I think WriteLineSeparator may be more intuitive than WriteRule.

Reply:
Change LGTM, this one is just a minor suggestion.